ports/espressif/boards/m5stack_cardputer/mpconfigboard.mk

```diff
@@ -0,0 +1,13 @@
+USB_VID = 0x303A
+USB_PID = 0x811A
```

Comment:
```suggestion
USB_PID = 0x816B
```

Reply:
If you take a look at the StampS3 [schematics](https://m5stack.oss-cn-shenzhen.aliyuncs.com/resource/docs/datasheet/Stamp/S007%20StampS3/Sch_M5StampS3_v0.2.pdf) you can see that is just a microcontroller with a USB port, a button and a RGB led. M5Stack use the Stamp S3 to make serveral different development board:

- [M5Capsule](https://docs.m5stack.com/en/core/M5Capsule)
- [M5Dial](https://docs.m5stack.com/en/core/M5Dial)
- [AirQ](https://docs.m5stack.com/en/core/AirQ)
- [Cardputer](https://docs.m5stack.com/en/core/Cardputer)

And problably more board are coming with the StampS3.
Each of these boards need a custom CircuitPython firmware, non a standard StampS3 CP firmware, therefore I suggest to use a dedicated USB PID for these boards.
There is already a pull request to add the USB PID for Cardputer: https://github.com/espressif/usb-pids/pull/135

Reply:
Thanks! I've updated the suggestion.